package.json

```diff
@@ -84,7 +85,7 @@
     "rollup-plugin-sucrase": "^2.1.0",
     "rollup-plugin-typescript": "^1.0.1",
     "rollup-plugin-virtual": "^1.0.1",
-    "source-map": "^0.6.1",
+    "source-map": "^0.7.3",
```

Comment:
Didn't source-map switch to an async API and that's why we've been having to stay on the old version? Or is it something else that's actually used to generate the sourcemaps that the compiler returns?

Reply:
It did, I updated the tests accordingly. I needed to upgrade because for a while the code generation was handled by astring, which needs source-map@0.7 to be installed alongside it. It's not used to generate the sourcemaps returned from the compiler, just for testing. No longer an issue, but figured we may as well keep it up to date now